comp/core/config/setup.go

```diff
@@ -12,16 +12,18 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/DataDog/viper"
+
```

Comment:
Why do we need viper here ?

Reply:
It's just my go linter which changed the import order (it was already there before this PR)